ding/reward_model/base_reward_model.py

```diff
@@ -90,7 +90,10 @@ def create_reward_model(cfg: dict, device: str, tb_logger: 'SummaryWriter') -> B
     cfg = copy.deepcopy(cfg)
     if 'import_names' in cfg:
         import_module(cfg.pop('import_names'))
-    reward_model_type = cfg.pop('type')
+    if hasattr(cfg, 'reward_model'):
```

Comment:
why use this branch, can you give me a specific case

Reply:
To use the cfg env, model info, we have to input cfg rather than cfg.reward_model